app/views/congress_messages/_form.html.erb

```diff
@@ -0,0 +1,43 @@
+<%= form_tag(congress_message_campaign_congress_messages_path(@campaign),
+             remote: true, id: "congress-message-create") do %>
+  <p class="step2-intro">
+    <%= @campaign.extra_fields_explain_text("We send your comments to your representatives by automatically filling out their online contact forms. Because each representative's form is a little different, we need your help filling out a few extra fields before we can send your message:") %>
+  </p>
+  <div id="errors"></div>
+  <% if @message.common_fields.any? %>
+    <h3 class="legislator-label common-fields">
+      <em>Common fields</em>
+    </h3>
+
+    <fieldset id="congressForms-common-fields">
+      <% @message.common_fields.each do |field| %>
+        <div class="form-group">
+          <%= congress_forms_field(field, @campaign, @message.common_attributes) %>
+        </div>
+      <% end %>
+    </fieldset>
+
+    <%= render "tools/newsletter_signup",
+               email: false, location: false, privacy_notice: false %>
+    <%= render "tools/save_my_info_option", info: %w(name address) %>
+  <% end %>
+
+  <% @message.forms_minus_common_fields.each do |form| %>
+    <fieldset class="congressForms-legislator-fields">
+      <h3 class="legislator-label">
+        <em><%= CongressMember.find_by(bioguide_id: form.bioguide_id).full_name %></em>
+      </h3>
+      <% form.fields.each do |field| %>
+        <div class="form-group">
+          <%= congress_forms_field(field, @campaign,
```

Comment:
I believe the new version is supposed to have a checkbox next to each legislator's name so they can choose who they submit to. I can try to add this in when this branch is ready if you like.